pin bump with better architecture

Gasoonjia opened the pull request:
Summary:
This diff bump torch pin to 1008.

also expose cuda version into torch_pin.py for better update if needed;

Differential Revision: D84280496

JacobSzwejbka reviewed (approved):
Review automatically exported from Phabricator review in Meta.

Gasoonjia committed:
pin bump with better architecture (#14957)

Summary:

This diff bump torch pin to 1008.

also expose cuda version into torch_pin.py for better update if needed;

Reviewed By: JacobSzwejbka

Differential Revision: D84280496
